src/text/text1.asm

```diff
@@ -1,306 +1,310 @@
 
-Text0007:
+RetreatText:
 	textfw "にげる"
 	done
 
-Text0008:
+WeaknessText:
 	textfw "弱点"
 	done
 
-Text0009:
+ResistanceText:
 	textfw "抵抗力"
 	done
 
-Text000a:
+PokemonPowerKanjiText:
+	; The formal term for Pokémon Power
+	; but was "PKMN PWR" in tcg1-en
 	textfw "特殊能力"
 	done
 
-Text000b:
+PokemonCardString: ; unused?
 	textfw "ポケモンカード"
 	done
 
-Text000c:
+LengthText:
 	textfw "身長"
 	done
 
-Text000d:
+WeightText:
 	textfw "体重"
 	done
 
-Text000e:
+PokemonText:
 	textfw "ポケモン"
 	done
 
-Text000f:
+LengthUnitMetresText:
 	textfw "m"
 	done
 
-Text0010:
+WeightUnitKilogramsText:
 	textfw "Kg"
 	done
 
-Text0011:
+PromostarRarityText:
 	textfw " "
 	done
 
-Text0012:
+CircleRarityText:
 	textfw "●"
 	done
 
-Text0013:
+DiamondRarityText:
 	textfw "◆"
 	done
 
-Text0014:
+StarRarityText:
 	textfw "★"
 	done
 
-Text0015:
+WhitestarRarityText:
 	textfw "☆"
 	done
 
-Text0016:
+AllCardsOwnedText:
 	textfw "のもっている全カード"
 	done
 
-Text0017:
+TotalNumberOfCardsText:
 	textfw "カードのごうけい         枚"
 	done
 
-Text0018:
+TypesOfCardsText:
 	textfw "カードのしゅるい         種"
 	done
 
-Text0019:
+GrassPokemonText:
 	textfw "草ポケモン"
 	done
 
-Text001a:
+FirePokemonText:
 	textfw "炎ポケモン"
 	done
 
-Text001b:
+WaterPokemonText:
 	textfw "水ポケモン"
 	done
 
-Text001c:
+LightningPokemonText:
 	textfw "雷ポケモン"
 	done
 
-Text001d:
+FightingPokemonText:
 	textfw "闘ポケモン"
 	done
 
-Text001e:
+PsychicPokemonText:
 	textfw "超ポケモン"
 	done
 
-Text001f:
+ColorlessPokemonText:
 	textfw "無色ポケモン"
 	done
 
-Text0020:
+TrainerCardText:
 	textfw "トレーナーカード"
 	done
 
-Text0021:
+EnergyCardText:
 	textfw "エネルギーカード"
 	done
 
-Text0022:
+DeckPrinterText:
 	textfw "デッキ"
 	done
 
-Text0023:
+AttackByText: ; unused?
 	textfw "のこうげき"
 	done
 
-Text0024:
+NoPokemonOnTheBenchText:
 	textfw "控えポケモンが いません"
 	done
 
-Text0025:
+UnableDueToSleepText:
 	textfw "「ねむり」状態なので できません"
 	done
 
-Text0026:
+UnableDueToParalysisText:
 	textfw "「マヒ」状態なので できません"
 	done
 
-Text0027:
+ReceivedNDamageDueToPoisonText:
 	textfw "<RAMTEXT>は"
 	linefw "「どく」状態なので <RAMNUM>ダメージ"
 	done
 
-Text0028:
+ReceivedNDamageDueToDoublePoisonText:
 	textfw "<RAMTEXT>は"
 	linefw "「どくどく」状態なので <RAMNUM>ダメージ"
 	done
 
-Text0029:
+IsStillAsleepText:
 	textfw "<RAMTEXT>は"
 	linefw "ねむり つづけている"
 	done
 
-Text002a:
+IsCuredOfSleepText:
 	textfw "<RAMTEXT>は"
 	linefw "ねむりから さめた"
 	done
 
-Text002b:
+IsCuredOfParalysisText:
 	textfw "<RAMTEXT>は"
 	linefw "マヒが とけた"
 	done
 
-Text002c:
+BetweenTurnsText:
 	textfw "ポケモンチェックをします"
 	done
 
-Text002d:
+UnableToUseItText:
 	textfw "つかえない状態です"
 	done
 
-Text002e:
+NoEnergyCardsText:
 	textfw "エネルギーカードが ありません"
 	done
 
-Text002f:
+IsThisOKText:
 	textfw "これで いいですか?"
 	done
 
-Text0030:
+YesOrNoText:
 	textfw "はい  いいえ"
 	done
 
-Text0031:
+DiscardActionName:
 	textfw "トラッシュする"
 	done
 
-Text0032:
+IncompleteText:
 	textfw "未完成"
 	done
 
-Text0033:
+DamageString: ; unused?
 	textfw "ダメージ"
 	done
 
-Text0034:
+UsedText:
 	textfw "<RAMTEXT>を"
 	linefw "つかいます"
 	done
 
-Text0035:
+ReceivedDamageText: ; unused?
```

Comment:
It would be pretty surprising if this was unused. Is it more likely that we just haven't disassembled the code that uses it yet? Or is there a different text string that is used for receiving damage?